src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -54,11 +55,14 @@ public class SwerveSubsystem extends SubsystemBase{
     private AHRS gyro = new AHRS(SPI.Port.kMXP);
     private SwerveDriveOdometry odometer = new SwerveDriveOdometry(DriveConstants.kDriveKinematics, new Rotation2d(0));
 
-    private DataLog datalog = DataLogManager.getLog();
-    private DoubleLogEntry translationXOutputLog = new DoubleLogEntry(datalog, "/swerve/txout"); //Logs x translation state output
-    private DoubleLogEntry translationYOutputLog = new DoubleLogEntry(datalog, "/swerve/tyout"); //Logs y translation state output
-    private DoubleLogEntry rotationOutputLog = new DoubleLogEntry(datalog, "/swerve/rotout"); //Logs rotation state output
-    private BooleanLogEntry controlOrientationLog = new BooleanLogEntry(datalog, "/swerve/orientation"); //Logs if robot is in FOD/ROD
+    private DataLog loopCountlog = DataLogManager.getLog();
+    private StringLogEntry loopCount = new StringLogEntry(loopCountlog, "/robot/loopCount");
+
+    private DataLog SwerveSubsystemdatalog = DataLogManager.getLog();
+    private DoubleLogEntry translationXOutputLog = new DoubleLogEntry(SwerveSubsystemdatalog, "/swerve/txout"); //Logs x translation state output
+    private DoubleLogEntry translationYOutputLog = new DoubleLogEntry(SwerveSubsystemdatalog, "/swerve/tyout"); //Logs y translation state output
+    private DoubleLogEntry rotationOutputLog = new DoubleLogEntry(SwerveSubsystemdatalog, "/swerve/rotout"); //Logs rotation state output
+    private BooleanLogEntry controlOrientationLog = new BooleanLogEntry(SwerveSubsystemdatalog, "/swerve/orientation"); //Logs if robot is in FOD/ROD
```

Comment:
These don't appear to get used anywhere?

Reply:
They are used, just not in the change list because the datalog was changed to SwerveSubsystemdatalog.